pyproject.toml

```diff
@@ -110,6 +110,7 @@ awadb = {version = "^0.3.3", optional = true}
 azure-search-documents = {version = "11.4.0a20230509004", source = "azure-sdk-dev", optional = true}
 # now streamlit requires Python >=3.7, !=3.9.7 So, it is commented out.
 #streamlit = {version = "^1.18.0", optional = true}
+pypandoc = {version = "^1.11", optional = true}
```

Comment:
this needs to get added to some optional group (eg `all`)

Reply:
I believe I have added it correctly in the latest commit. Working on the example notebook now!